numbert/utils/utils_numbert.py

```diff
@@ -229,100 +231,147 @@ def merge(self, qrels, run, queries):
             data[query_id] = (query, relevant_doc_ids, candidate_doc_ids)
         return data
 
-    def get_train_examples(self, data_dir, is_qrels = None):
+    def get_train_examples(self, data_dir, is_qrels = None, is_duoBERT = False):
         """See base class."""
 
         return self._create_examples_train_triples(self.load_train_triples(
                 os.path.join(data_dir,"triples.train.small.tsv")),
-                "train")
+                "train", is_duoBERT)
 
-    def get_train_examples_old(self, data_dir, is_qrels = True):
+    def get_train_examples_old(self, data_dir, is_qrels = True, is_duoBERT = False):
```

Comment:
What is this used for? Maybe rename it `_v1`or `deprecated`?

Reply:
This is for using the other train file provided by MSMARCO instead of the triples one. I'll rename it as  *_non_triples. Right now this isn't being used but potentially can use this in the future.